cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -74,15 +74,15 @@ spec:
             - >
               LIVENESS_THRESHOLD_SECONDS=${LIVENESS_THRESHOLD_SECONDS:-300};
               STUCK_THRESHOLD_SECONDS=${LIVENESS_THRESHOLD_SECONDS:-900};
-              if [ ! -e /var/log/fluentd-buffers ];
-              then
-                exit 1;
-              fi;
-              LAST_MODIFIED_DATE=`stat /var/log/fluentd-buffers | grep Modify | sed -r "s/Modify: (.*)/\1/"`;
+              POS_FILE=${POS_FILE:-/var/log/gcp-containers.log.pos}
```

Comment:
Mb replace with taking the latest timestamp of all among `.pos` files by default?